src/main/java/uk/nhs/digital/emailadapter/service/StagedStopwatch.java

```diff
@@ -1,17 +1,16 @@
-package net.nhs.sgh.emailadapter.service;
+package uk.nhs.digital.emailadapter.service;
 
 import java.time.Instant;
 import java.time.temporal.ChronoUnit;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
+import lombok.extern.slf4j.Slf4j;
 
+@Slf4j
 public class StagedStopwatch {
 
-  private static final Logger logger = LoggerFactory.getLogger(StagedStopwatch.class);
   private Instant stageStart = Instant.now();
 
   private StagedStopwatch() {
-    logger.info("Timing started at " + stageStart + '\n');
+    log.info("Timing started at " + stageStart + '\n');
```

Comment:
use parameterized version of logging stmts everywhere and avoid unnecassary parameter calculation if the logging level is lower.

log.info(".... {} ", stageStart);